oci-library/oci-hol/java-management/set-up-logrotate/set-up-logrotate.md

```diff
@@ -0,0 +1,138 @@
+# Set up logrotate
+
+## Introduction
+
+This lab will guide you in setting up the logrotate utility on your host machine to periodically rotate the `usagetracker.log` generated by Java Usage Tracker.
+
+Although the size of the Java Usage Tracker log file is small, you should consider periodically truncate, compress, archive, or delete the log file. This is because the Java Usage Tracker does not check for available disk space or perform these administrative tasks on the log file as it incrementally adds records to it. 
+
+Estimated Time: 10 minutes
+
+### Objectives
+
+In this lab, you will:
+
+* Set up logrotate on Linux instances
+* Set up logrotatewin on Windows instances
+
+### Prerequisites
+
+* Access to your host machine with management agent already setup
+* Completed labs 1 to 4
+
+## Task 1: Set up logrotate on Linux instances
+
+1. Run the following two commands:
+
+    ```
+    <copy>
+    sudo yum install logrotate
+    sudo touch /etc/logrotate.d/jms
+    </copy>
+    ```
+
+2. Open the file /etc/logrotate.d/jms in a text editor: 
+
+    ```
+    <copy>
+    sudo nano /etc/logrotate.d/jms
+    </copy>
+    ```
+
+3. Paste the following into the file:
+
+    ```
+    <copy>
+    /var/log/java/usagetracker.log {
+      su mgmt_agent root
+      create 666 mgmt_agent root
+      rotate 5
+      size 20M
+      notifempty
+      compress
+      missingok
+    }
+    </copy>
+    ```
+
+4. To save the file, press CTRL+x. Before exiting, nano will ask you if you wish to save the file: Type y to save and exit, type n to abandon your changes and exit.
+
+## Task 2: Set up logrotatewin on Windows instances
```

Comment:
Tested on Windows, it works. One point to note for troubleshooting, SCHTASKS and other commands made use of, may need a C:\Windows\system32 if they haven't already added to their PATH

Reply:
ok this is interesting, something new I learnt today. How did you know about this?

Reply:
Because if the user / system doesn't already set up the paths, everything in the C:\Windows\system32 folder they cannot access. You will find a lot of ppl google, that command not found, it is because the "shortcut" isn't set up. Hence they either set up the shortcut permanently, or they use add C:\Windows\system32 to path variable when running commands. 

I know about this because the batch file for Mariott Amit created, then when testing, I modified it, this was one of the initial issues why it couldn't run.